src/screens/FeedScreen/Content.js

```diff
@@ -251,11 +250,7 @@ const Content = ({
       ) : null}
 
       {item && item.post_type === POST_TYPE_POLL ? (
-        <View
-          style={[
-            styles.containerMainText(handleContainerText().isShort),
-            {marginVertical: handleMarginVertical(message)}
-          ]}>
+        <View style={styles.containerMainText(handleContainerText().isShort)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the View container for the poll content has been simplified. The marginVertical style property which was previously dynamically calculated by `handleMarginVertical(message)` has been removed. Make sure this doesn't affect the layout of your component adversely.
```diff
-        <View
-          style={[
-            styles.containerMainText(handleContainerText().isShort),
-            {marginVertical: handleMarginVertical(message)}
-          ]}>
+        <View style={styles.containerMainText(handleContainerText().isShort)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->